osm2pgsql.lua

```diff
@@ -499,130 +499,129 @@ function osm2pgsql.process_node(object)
         object.tags.wikidata == 'Q994730') then
         output_hstore['disputed_by'] = 'CN;RU;IN;GR'
     end
--- Recast Northern Cyprus as country label and turn off for several POVs including China and Russia
+
+-- Turn off Northern Cyprus label for most countries
     if object.tags.place and object.tags.wikidata == 'Q23681' then
-        output_hstore['place'] = 'country'
+        output_hstore['place'] = 'region'
```

Comment:
So this needs to be `region` or `country` (region is better than country) to get included by filters in places.yaml, but then most countries shouldn't see this label, but Turkey should. 

Something like?

> output_hstore['place:TR'] = 'country'

Reply:
Another option is we extend places.yaml to filter also on **place** = `unrecognized` for countries, that would make all this easier to follow.

Reply:
place=unrecognized sounds like a good idea. It will remove some points of failure in the process and simplify this. Going to give that try

Reply:
This seems to be working nicely, let's see it in a build to 100% check :)